proteomic_aggregation_and_visualization/python/proteomic_aggregation_and_visualization.ipynb

```diff
@@ -0,0 +1,4004 @@
+{
```

Comment:
<p>I wonder if it would simplfy things a lot if we just took the annotation from the Protein Report.  I understand this isn't the "perfect" way, but it's they way er </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>